### PR TITLE
Updating functionaltest.ps1 so it works in VSTS.

### PR DESCRIPTION
**Pull request by pradipd**

### CODE REVIEW

**Comment by pradipd** [comment]
@jaredhms is added to the review. #Closed

**Comment by pradipd** [comment]
@mukhole is added to the review. #Closed

File: tests/Run-FunctionalTests.ps1 [hunk 1]
```diff
@@ -176,7 +182,31 @@ function ExecTest($argList)
...
+                    write-host "##[error]: $o" -foregroundcolor red
+
+                } else {
+                    write-host $o
+                }
+            }
+        } else {
+            Invoke-Expression "$taefPath $arguments"
+            $script:exitCode = $LASTEXITCODE
+        }
```
Comment (pradipd) [inline tests/Run-FunctionalTests.ps1]: Brian should just be able to take this and add it to his pull request so we can have the UT's running TAEF as well.

File: tests/Run-FunctionalTests.ps1 [hunk 2]
```diff
@@ -176,7 +182,31 @@ function ExecTest($argList)
     else
     {
         $arguments = "$testPath" + "$argList"
-        Invoke-Expression "$taefPath $arguments"
+        if($RedirectTAEFErrors) {
```
Comment (jaredhms) [inline tests/Run-FunctionalTests.ps1]: >( [](start = 10, length = 1)

nit: missing spaces - 'if (', etc.

**Comment by jaredhms** [comment]
:shipit:

**Pull request merged**